admin/schedule-form-excludes.php

```diff
@@ -298,12 +298,15 @@
 							$size = $site_size->filesize( $file );
 
 							if ( false !== $size ) :
-								$size          = $size ?: '0';
-								$excluded_size = $excluded_site_size->filesize( $file ) ?: '0'; ?>
+								$size          = $size ?: 0;
+								$excluded_size = $excluded_site_size->filesize( $file ) ?: 0;
```

Comment:
Actually, I think it should be this function returning `0` for excluded files, so that down the line they get sorted to the bottom of the list of sizes. I'll double check if this is used anywhere else.